README.md

```diff
@@ -52,6 +53,11 @@ python setup.py install
     Runs the acutal migration. Will go through, the database table by table
     and migrate all rows to the target database.
 
+    It is important to note that the tool is assuming there is nothing new being
+    written to the source database during migration or once migrated. If
+    a rerun of the migration is required, it is recommended that the target
+    tables have been first emptied of rows.
```

Comment:
AFAIU, target tables are emptied by the tool when the migration command is run. So we don't need to tell users anything?

Reply:
Truth is, @rhafer just came with the idea that we should offer a specific command just for purging the tables, but that will have it's own PR